impl/src/bitfield.rs

```diff
@@ -180,6 +184,56 @@ impl BitfieldStruct {
         ))
     }
 
+    /// Get a list of fields, both in the form of a `String` as well as the corresponding faillible
+    /// getter method
+    fn get_field_strings_and_getters(&self) -> (Vec<String>, Vec<syn::Ident>) {
+        self.item_struct
+            .fields
+            .iter()
+            .enumerate()
+            .map(|(i, field)| {
+                let field_name = field
+                    .ident
+                    .as_ref()
+                    .map(syn::Ident::to_string)
+                    .unwrap_or_else(|| format!("get_{}", i));
+                let fallible_getter =
+                    quote::format_ident!("{}_or_err", field_name, span = field.span());
```

Comment:
This is not the correct getter identifier for tuple structs with unnamed fields.
For those it is: `get_N_or_err` where `N` denotes the index of the field.

Reply:
Ah this was a mistake of me trying to get too fancy when refactoring some stuff. Thanks!